### PR TITLE
trivial: fix ultra96 serial macro

### PR DESCRIPTION
**Pull request by chrisguikema**

### CODE REVIEW

**Comment by chrisguikema** [comment]
``` 
chrisguikema: ~/seL4/UPSTREAM/sel4test/test  $ ag CONFIG_PLAT_ZYNQMP       
kernel/gen_config/kernel/gen_config.h
23:#define CONFIG_PLAT_ZYNQMP_ULTRA96V2  1  /* KernelPlatformZynqmpUltra96v2=ON */
```

The macro doesn't have a space when compiling for the ultra96. Not sure how that happened.

**Comment by axel-h** [comment]
What do you mean by "space"?

**Review by axel-h — APPROVED** [review]
Looks ok to me. 

Maybe we should make the unconditional `#else` raise an error and have board specific checks only?

**Comment by chrisguikema** [comment]
Yes, underscore.

File: libplatsupport/CMakeLists.txt [hunk 1]
```diff
@@ -134,7 +134,7 @@ if(KernelPlatImx8mq OR KernelPlatformImx8mm-evk)
     target_include_directories(platsupport PUBLIC plat_include/imx8m)
 endif()
 
-if(KernelPlatformZynqmpZcu102 OR KernelPlatformZynqmpUltra96)
+if(KernelPlatformZynqmpZcu102 OR KernelPlatformZynqmpUltra96 OR KernelPlatformZynqmpUltra96v2)
```
Comment (axel-h) [inline libplatsupport/CMakeLists.txt:137]: We could also use `if(KernelPlatformZynqmp)` here, it would be a step towards unification.

Comment (lsf37) [reply libplatsupport/CMakeLists.txt:137]: Let's leave that one for a separate PR.

**Review by Ivan-Velickovic — APPROVED** [review]

**Review by lsf37 — APPROVED** [review]

**Comment by axel-h** [comment]
@lsf37 Is this good to merge? Or do you want to add the Ultra96 to the HW builds also?

**Comment by lsf37** [comment]
Fine to merge from my side.

**Review by kent-mcleod — APPROVED** [review]

**Pull request merged**